src/utils/calculateDiffPayments.util.ts

```diff
@@ -0,0 +1,47 @@
+export function calculateDiffPayments(
+  sum: number,
+  monthsToPay: number,
+  percentage: number,
+  monthsArray: string[]
+): Array<any> {
+  const finalResult = [];
+  let totalBasicDept = sum;
+  const basicDebtPeymentPerMonth = sum / monthsToPay;
+  const currentDay = new Date();
```

Comment:
not important, we could discuss it, but I think would be better to have a type while declaring new variables.